addons/web_editor/models/ir_qweb.py

```diff
@@ -79,11 +79,16 @@ def _compile_directive_snippet(self, el, options, indent):
         view_id = View.get_view_id(key)
         name = View.browse(view_id).name
         thumbnail = el.attrib.pop('t-thumbnail', "oe-thumbnail")
-        div = '<div name="%s" data-oe-type="snippet" data-oe-thumbnail="%s" data-oe-snippet-id="%s" data-oe-keywords="%s">' % (
+        # Forbid sanitize contains the specific reason:
+        # - "true": always forbid
+        # - "form": forbid if forms are sanitized
+        forbid_sanitize = el.attrib.get('t-forbid-sanitize')
+        div = '<div name="%s" data-oe-type="snippet" data-oe-thumbnail="%s" data-oe-snippet-id="%s" data-oe-keywords="%s" %s>' % (
             escape(pycompat.to_text(name)),
             escape(pycompat.to_text(thumbnail)),
             escape(pycompat.to_text(view_id)),
-            escape(pycompat.to_text(el.findtext('keywords')))
+            escape(pycompat.to_text(el.findtext('keywords'))),
+            f'data-oe-forbid-sanitize="{forbid_sanitize}"' if forbid_sanitize else '',
```

Comment:
why not do the same semantic as for `data-oe-snippet-id` ?

`... data-oe-keywords="%s" data-oe-forbid-sanitize="%s">`

Reply:
Because it is a very rare attribute (3~4 occurrences) - therefore we thought it should not appear unless it is actually used.